src/pybind/mgr/devicehealth/module.py

```diff
@@ -9,6 +9,7 @@
 import rados
 from threading import Event
 from datetime import datetime, timedelta, date, time
+from six import iteritems
```

Comment:
six is not a builtin module. we don't list python-six as a dependency. and i don't think it's worthwhile to pull it in as one.

Reply:
Good point -- I had somehow assumed we were already using it, but it certainly isn't worth pulling in for this one case.

I'm kind of surprised we haven't already run into more situations that would use it.

Reply:
@theanalyst i am asking @LenzGr 's opinion on adding python-six as a dependency of ceph.

Reply:
ack, then I'll revert to the python six version again :)

Reply:
reverted